File: python/day01/part1.py
from file_reader import read_file

lines = read_file('input.txt', '\n\n')
elf_calories = [list(map(int, line.strip().split('\n'))) for line in lines]

max = 0
for elf in elf_calories:
    if max <= sum(elf):
        max = sum(elf)
print(max)

File: javascript/day01/part1.js
const { assert } = require('console')
const input = require('../file-reader.js').readFile('input.txt', '\n', Number, v => true)

let max = 0
let total = 0
input.forEach(val => {
    if (val === 0) {
        max = max < total ? total : max
        total = 0
    } else {
        total += val
    }
});

console.log(max)
assert(max === 69693, `Incorrect result`)
